aikau/src/main/resources/alfresco/core/ResizeMixin.js

```diff
@@ -61,7 +61,9 @@ define(["dojo/_base/declare",
          // Fire a custom event to let contained objects know that the node has been resized.
          this.throttledPublish(this.alfResizeNodeTopic, {
             node: resizedNode
-         }, true);
+         }, true, false, {
+            timeoutMs: 100
```

Comment:
Not sure that makes a lot of sense here? This is essentially a black-box event where you're saying "I don't want resize-events to happen any more often than 100ms", and it feels like an arbitrary but intentional number that can safely be chosen by us, and seems especially appropriate as it coincides with the interval where we're checking nodes for size-changes.